docs/src/cheat-sheets.md

```diff
@@ -8,11 +8,6 @@ development purposes.
 
 ### Build
 
-```bash
-# Build all packages in the repository
-cargo build --workspace
```

Comment:
Should this be kept with `--workspace` removed?

Reply:
I'm open to other ideas but IMO it was only ever in there in the first place to note that `--workspace` was needed.